lib/server/utils.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2025 Edgecast Cloud LLC.
+ */
+
+/**
+ * Utility functions for Mahi
+ */
+
+/**
+ * Create hexdump output (like xxd)
+ */
+function hexdump(str) {
```

Comment:
Phew.  This is adding new functionality, not a whole new executable like I saw before.